Dockerfile.prod

```diff
@@ -17,18 +17,14 @@ RUN echo '{ "allow_root": true }' > /root/.bowerrc && \
 # install missing stuff
     apk add --no-cache --virtual .gyp python make g++ git && \
 
-# install npm globals
-    npm install -g bower && \
```

Comment:
previously `npm run setup` tend throw in case of missing bower, is it taking the local one now?

Reply:
https://github.com/Wikia/mobile-wiki/pull/240/files#diff-0e9722a692af2b244674b56146a1ecd3R26

Reply:
and when scripts are run via "npm run" they have .bin added to path